hermes-mock/src/main/java/pl/allegro/tech/hermes/mock/HermesMock.java

```diff
@@ -0,0 +1,210 @@
+package pl.allegro.tech.hermes.mock;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.ResponseDefinitionBuilder;
+import com.github.tomakehurst.wiremock.verification.LoggedRequest;
+import org.apache.avro.AvroRuntimeException;
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericDatumReader;
+import org.apache.avro.generic.GenericDatumWriter;
+import org.apache.avro.io.Decoder;
+import org.apache.avro.io.DecoderFactory;
+import org.apache.avro.io.Encoder;
+import org.apache.avro.io.EncoderFactory;
+import org.apache.http.HttpStatus;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.aResponse;
+import static com.github.tomakehurst.wiremock.client.WireMock.get;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathMatching;
+import static com.jayway.awaitility.Awaitility.await;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static java.util.stream.Collectors.toList;
+
+public class HermesMock {
+    private WireMockServer wireMockServer;
+    private ObjectMapper objectMapper = new ObjectMapper();
+    private int awaitSeconds = 5;
+
+    public HermesMock() {
+        wireMockServer = new WireMockServer();
+    }
+
+    public HermesMock(int port) {
+        wireMockServer = new WireMockServer(port);
+    }
+
+    public void setAwaitSeconds(int awaitSeconds) {
+        this.awaitSeconds = awaitSeconds;
+    }
+
+    public void setObjectMapper(ObjectMapper objectMapper) {
```

Comment:
I would prefer to configure `HermesMock` instance at creation time, so no setters. Sth like

```java
HermesMock my = HermesMock.builder().port(...).objectMapper(...).build();
```